config/main.py

```diff
@@ -2567,12 +2567,18 @@ def synchronous_mode(sync_mode):
 #
 @config.command('suppress-fib-pending')
 @click.argument('state', metavar='<enabled|disabled>', required=True, type=click.Choice(['enabled', 'disabled']))
+@click.option('-n', '--namespace', help='Namespace name', required=True if multi_asic.is_multi_asic() else False,
```

Comment:
@vganesan-nokia I think we should not give namespace option to user during config.. Instead, for multi-asic, we should just enable on all Asics? 
Cant think of a usecase, where user would want to enable this feature on one asic and not on other.. 

Otherwise, this feature would get very complicated to test on multi-asic systems.

Reply:
O.k. Thanks @deepak-singhal0408 for the feedback. We can have the "namespace" option but as an optional one instead of required. With this In multi-asic system, if the name space is not given, the configuration will be applicable for all the asics in the system. Or, do we want this "--namespace" option to be completely removed?

Reply:
I would prefer the --namespace option to be completely removed from config (otherwise someone might use that and we will have inconsistent config across asics).. We can just make changes to configure on all asics when this command is executed..
In show command also, I would suggest to remove --namespace option.. just display for all asics on multi-asic systems..